src/containers/InfoProyecto/index.js

```diff
@@ -39,134 +85,198 @@ export default function InfoProyecto({ projectData, type, state }) {
-          </Box>
+            <Box mt={2}>
+              <Typography
+                style={{ fontWeight: 600 }}
+                variant="body1"
+                display="inline"
+                gutterBottom
+              >
+                {PROYECT_FORM_LABELS.TIPO_PROYECTO}:{" "}
+              </Typography>
+              <Typography display="inline" variant="body1">
+                {type}
+              </Typography>
+            </Box>
 
-          <Box mt={2} style={{ display: "flex", gap: "0.5rem" }}>
-            <Typography
-              style={{ fontWeight: 600 }}
-              variant="body1"
-              display="inline"
-              gutterBottom
-            >
-              Estado:{" "}
-            </Typography>
-            {renderColor(state)}
-          </Box>
+            <Box mt={2} style={{ display: "flex", gap: "0.5rem" }}>
+              <Typography
+                style={{ fontWeight: 600, alignSelf: "center" }}
+                variant="body1"
+                display="inline"
+                gutterBottom
+              >
+                {PROYECT_FORM_LABELS.ESTADO}:{" "}
+              </Typography>
+              <FormControl>
+                <Select
+                  labelId="state-input"
+                  label="Estado"
+                  id="project_state"
+                  name="project_state"
+                  style={{ width: "unset" }}
+                  value={project_state}
+                  onChange={(e) => {
+                    setProjectState(e.target.value);
+                  }}
+                >
+                  <MenuItem value="verde">{renderColor("verde")}</MenuItem>
+                  <MenuItem value="amarillo">
+                    {renderColor("amarillo")}
+                  </MenuItem>
+                  <MenuItem value="rojo">{renderColor("rojo")}</MenuItem>
+                  <MenuItem value="upcoming">
+                    {renderColor("upcoming")}
+                  </MenuItem>
+                </Select>
+              </FormControl>
+            </Box>
 
-          <Box mt={2}>
-            <Typography
-              style={{ fontWeight: 600 }}
-              variant="body1"
-              display="inline"
-              gutterBottom
-            >
-              Fecha Inicio:{" "}
-            </Typography>
-            <Typography display="inline" variant="body1">
-              {moment(projectData.start_date).format("DD/MM/YYYY")}
-            </Typography>
-          </Box>
-          <Box mt={2}>
-            <Typography
-              style={{ fontWeight: 600 }}
-              variant="body1"
-              display="inline"
-              gutterBottom
-            >
-              Fecha Fin:{" "}
-            </Typography>
-            {projectData.end_date ? (
+            <Box mt={2}>
+              <Typography
+                style={{ fontWeight: 600 }}
+                variant="body1"
+                display="inline"
+                gutterBottom
```

Comment:
This is kind of duplicated. You can create a (styled) component which extends from Typography and set this styles by default.
Then use the new component wherever you want.